project/project.ts

```diff
@@ -356,6 +356,10 @@ export const makeTsGdProject = async (
           // Chokidar is inconsistent about whether it passes in stats or not -
           // sometimes it does and sometimes it doesn't.
 
+          if (!fs.existsSync(path)) {
```

Comment:
Do we need to additionally handle clean up somehow? E.g. some unlink event handler?
I think we might have a reference to previously exisiting files stored in the onAdd handler.

Reply:
You are right, there is `unlink` handler which should be invoked. I changed it to not loading `stats` when file does not exist.